components/ai_chat/distiller_scripts/BUILD.gn

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/common/typescript.gni")
+import("//tools/grit/repack.gni")
+
+transpile_web_ui("ai_chat_site_distiller") {
+  resource_name = "ai_chat_site_distiller"
```

Comment:
this should have `visibility =  [ ":*" ]`
I'm a bit confused about why it's using `transpile_web_ui` if these aren't used as part of a webui. At the very least this is confusing, but if this target is for generalized webpack and not specific to webui then it should probably have a different name cc @petemill

Reply:
also please use consistent naming (`distiller_scripts` vs `ai_chat_site_distiller`)

Reply:
I went ahead and added explicit _visibility_. Regarding the use of `transpile_web_ui`, this appears to be the only route to transpiling TypeScript files, AFAIK. It's the route used by our _creator detection_ and _cosmetic filter_ scripts, for example.